.github/workflows/ci.yml

```diff
@@ -144,6 +144,39 @@ jobs:
         MATRIX_CONTEXT: ${{ toJson(matrix) }}
       if: ${{ github.event_name == 'push' && failure() }}
 
+  build-and-test-release-macos:
+    name: Run Tests (Release)
```

Comment:
nit: we should probably start putting the platform name to the name of the jobs, such that we know what's going on.

Reply:
Maybe we can have a name scheme like:
``
Test iOS latest (Debug)
Test iOS latest (Release)
Test iOS 13.5 (Debug)
Test iOS 13.5 (Release)
Test iOS 12.3 (Debug)
Test iOS 12.3 (Release)
Test macOS latest (Debug)
Test macOS latest (Release)
``
cc @b-onc ?

Reply:
Missed name property, my bad

Reply:
Looks like we will need quite a few jobs to test everything properly 🤔